core/src/main/scala/org/apache/spark/ui/jobs/JobsTab.scala

```diff
@@ -16,9 +16,9 @@
  */
 
 package org.apache.spark.ui.jobs
-
+import javax.servlet.http.HttpServletRequest
```

Comment:
Hm, I'm not sure if this back-port is correct. This file's change doesn't look like it does anything and I don't see this change in the original: https://github.com/apache/spark/pull/17686/files

Reply:
Will look into this...

Reply:
Agreed, will remove.